build-tools/aapt2.cmake

```
@@ -0,0 +1,198 @@
+# ========================= aapt2 proto ============================
+set(AAPT2_PROTO_SRC)
+set(AAPT2_PROTO_HDRS)
+set(AAPT2_PROTO_DIR ${SRC}/base/tools/aapt2)
+
+file(GLOB_RECURSE PROTO_FILES ${AAPT2_PROTO_DIR}/*.proto)
+
+foreach(proto ${PROTO_FILES})
+    get_filename_component(FIL_WE ${proto} NAME_WE)
+
+    if(DEFINED PROTOC_PATH)
+        # execute the protoc command to generate the proto targets for host arch
```

```suggestion
        # Execute the protoc command to generate the proto targets for host arch.
```

Use this comment style.